tests/Microsoft.DotNet.XHarness.iOS.Tests/ErrorKnowledgeBaseTests.cs

```diff
@@ -113,5 +113,35 @@ public void UsbIssuesMissintTest()
                 File.Delete(logPath);
             }
         }
+
+        [Fact]
+        public void DeviceLockedTest()
+        {
+            var expectedFailureMessage = "Cannot launch the application because the device is locked. Please unlock the device and try again";
```

Comment:
minor idea: If test log was in a string constant we would avoid bunch of log.write and it would be more readable.